src/fides/api/models/privacy_notice.py

```diff
@@ -434,18 +425,35 @@ class PrivacyNoticeHistory(NoticeTranslationBase, PrivacyNoticeBase, Base):
     """
 
     origin = Column(
-        String, ForeignKey(PrivacyNoticeTemplate.id_field_path), nullable=True
+        String,
+        ForeignKey(PrivacyNoticeTemplate.id_field_path),
     )  # pointer back to the PrivacyNoticeTemplate
 
     translation_id = Column(
-        String, ForeignKey(NoticeTranslation.id_field_path, ondelete="SET NULL")
-    )  # pointer back to the NoticeTranslation
+        String,
+        ForeignKey(NoticeTranslation.id_field_path, ondelete="SET NULL"),
+        index=True,
+    )  # pointer back to the NoticeTranslation.  Set to null if the translation is deleted, but
+    # we retain this record for consent reporting
 
     version = Column(Float, nullable=False, default=1.0)
 
-    privacy_notice_id = Column(
-        String, ForeignKey(PrivacyNotice.id_field_path), nullable=True
-    )  # TODO Will be removed.  This now points to just the translation.
+    # Where a notice is displayed is now configured on the Experience side!
+    displayed_in_privacy_center = Column(
```

Comment:
nit: would it be worth throwing this into a base class of deprecated fields so it's easier to keep track of moving forward? i think you've done something like that before. may not be worth the effort here though!

Reply:
you're right that would be clearer!